tests/attr/layer/test_layer_integrated_gradients.py

```diff
@@ -0,0 +1,109 @@
+#!/usr/bin/env python3
+
+import torch
+
+from ..helpers.utils import assertArraysAlmostEqual
+
+from captum.attr._models.base import (
+    configure_interpretable_embedding_layer,
+    remove_interpretable_embedding_layer,
+)
+
+from captum.attr._core.integrated_gradients import IntegratedGradients
+from captum.attr._core.layer.layer_integrated_gradients import LayerIntegratedGradients
+from captum.attr._core.layer.layer_conductance import LayerConductance
+
+from ..helpers.utils import BaseTest
+from ..helpers.basic_models import (
+    BasicEmbeddingModel,
+    BasicModel_MultiLayer,
+)
+
+
+class Test(BaseTest):
+    def test_compare_with_emb_patching(self):
+        input1 = torch.tensor([[2, 5, 0, 1]])
+        baseline1 = torch.tensor([[0, 0, 0, 0]])
+        # these ones will be use as an additional forward args
+        input2 = torch.tensor([[0, 2, 4, 1]])
+        input3 = torch.tensor([[2, 3, 0, 1]])
+
+        self._assert_compare_with_emb_patching(
+            input1, baseline1, additional_args=(input2, input3)
+        )
+
+    def test_compare_with_emb_patching_batch(self):
+        input1 = torch.tensor([[2, 5, 0, 1], [3, 1, 1, 0]])
+        baseline1 = torch.tensor([[0, 0, 0, 0]])
+        # these ones will be use as an additional forward args
+        input2 = torch.tensor([[0, 2, 4, 1], [2, 3, 5, 7]])
+        input3 = torch.tensor([[3, 5, 6, 7], [2, 3, 0, 1]])
+
+        self._assert_compare_with_emb_patching(
+            input1, baseline1, additional_args=(input2, input3)
+        )
+
+    def test_compare_with_layer_conductance_attr_to_outputs(self):
+        model = BasicModel_MultiLayer()
+        input = torch.tensor([[50.0, 50.0, 50.0]], requires_grad=True)
+        self._assert_compare_with_layer_conductance(model, input)
+
+    def test_compare_with_layer_conductance_attr_to_inputs(self):
+        model = BasicModel_MultiLayer()
+        input = torch.tensor([[50.0, 50.0, 50.0]], requires_grad=True)
+        self._assert_compare_with_layer_conductance(model, input, True)
+
+    def _assert_compare_with_layer_conductance(
+        self, model, input, attribute_to_layer_input=False
+    ):
+        lc = LayerConductance(model, model.linear2)
```

Comment:
nit: It might be nice to have a comment here explaining that although they are not theoretically equivalent, they are both integrating along paths between the same 2 points, and with only linear transformations prior to the layer they are equivalent? It might be confusing that this property should hold in general.